icicle/primitives/field.cuh

```diff
@@ -37,6 +38,7 @@ class Field
 public:
   static constexpr unsigned TLC = CONFIG::limbs_count;
   static constexpr unsigned NBITS = CONFIG::modulus_bit_count;
+  static constexpr unsigned TWO_ADICITY = CONFIG::omegas_count;
```

Comment:
Did we mean to use this in the if statement conditions on line 61,71, and 83?

Reply:
No not really (though we can use it there too). This just needs to be public for the ntt to know what sizes it can support